spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -768,7 +741,6 @@
             expect(job_analytics).to have_logged_event(
               'GetUspsProofingResultsJob: Unexpected response received',
               hash_including(
-                minutes_since_established: range_approximating(3.days.in_minutes, vary_right: 5),
                 reason: 'Invalid applicant unique id',
```

Comment:
```suggestion
                reason: 'Invalid applicant unique id',
                response_message: /Applicant [0-9a-z]{18} does not exist/,
```
could also test for response message here